helm/dagster/schema/schema/charts/dagster/subschema/retention.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any, Dict, List, Optional, Union
+
+from pydantic import BaseModel, Extra
+
+
+class TickRetentionByType(BaseModel):
+    skipped: Optional[int]
+    success: Optional[int]
+    failure: Optional[int]
+    started: Optional[int]
+
+    class Config:
+        extra = Extra.forbid
+
+
+class TickRetention(BaseModel):
+    purgeAfterDays: Union[int, TickRetentionByType]
```

Comment:
It's not intuitive that this is a union. I don't think it's clear on https://docs.dagster.io/deployment/dagster-instance#data-retention either.